docs/src/ActivationCard.doc.js

```diff
@@ -158,4 +158,136 @@ card(
   />,
 );
 
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Custom navigation"
+      description={`
+These examples illustrate a custom navigation implementation to externally control the link functionality within ActivationCard.
+
+If passed to Provider's \`onNavigation\` prop, \`useCustomOnNavigationProvider\`, a high-order function, is passed down to ActivationCard where it's executed. Then, \`onNavigation\` returns a function that gets called during the \`onClick\` event handler.
```

Comment:
nit: "higher-order" (here and throughout)

Reply:
DONE